paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -392,12 +392,24 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
     std::shared_ptr<mkldnn::convolution_forward::primitive_desc> conv_pd;
     std::shared_ptr<platform::ConvMKLDNNHandler> handler;
 
-    auto prim_key = key + "@conv_p";
-    auto dst_key = key + "@dst_mem_p";
-    auto src_key = key + "@src_mem_p";
-    auto user_src_key = key + "@user_src_mem_p";
-    auto src_reorder_key = key + "@src_mem_preorder_p";
-    auto residual_reorder_key = key + "@residual_data_mem_preorder_p";
+    // This is workaround for hacky implementation
+    // of conv int8 mkl-dnn. Once conv fp32 and conv int8
+    // are merged/unified, this will disappear
+    std::string key_tid = "";
+    if (platform::get_cur_mkldnn_session_id() ==
+        platform::kMKLDNNSessionID_Default) {
+      auto tid = std::this_thread::get_id();
+      std::stringstream ss;
+      ss << tid;
+      key_tid = "-t:" + ss.str();
```

Comment:
Wouldn't it be enough to simplify 
```
std::stringstream ss;
ss << tid;
key_tid = "-t:" + ss.str();
```
down to:
`key_tid = "-t:" + std::to_string(tid);`

Reply:
Thanks for suggestion. This is very elegant proposal. I'm not planning to change it in that PR as this code will be removed once @lidanqing-intel finish her unification task of convolution. But I have similar section of code inside of MKLDNNHandler , so I can experiment with your proposal in next PR